Detectors/Vertexing/include/DetectorsVertexing/PVertexer.h

```diff
@@ -37,7 +37,7 @@
 
 //TODO: MeanVertex and parameters input from CCDB
 
-//#define _PV_DEBUG_TREE_ // if enabled, produce dbscan and vertex comparison dump
+#define _PV_DEBUG_TREE_ // if enabled, produce dbscan and vertex comparison dump
```

Comment:
@shahor02  Just be careful here, I think this is a leftover

Reply:
Thanks,indeed, need to comment this out

Reply:
Why don't you use directly the misalignment from the configurable param to compute the sigmas, applying some default value if it is empty?